presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -109,6 +113,26 @@ Operations
 
     Returns the geometry value that represents the point set symmetric difference of two geometries.
 
+.. function:: ST_Buffer(Geometry, double) -> Geometry
+
+    Returns the geometry value that represents all points whose distance from any point of an input geometry is less than or equal to a specified distance
+
+.. function:: point_to_bing_tile(Geometry, int) => varchar
+
+    Given a (longitude, latitude) point and a zoom level, returns a QuadKey for the containing Bing tile. Latitude must be within [-85.05112878, 85.05112878] range. Longitude must be within [-180, 180] range. Zoom levels from 1 to 23 are supported.
```

Comment:
Use code formatting for things ranges:
```
``[-180, 180]``
```
Also, hard wrap text to around 80-100 characters (see the rest of the documentation). This file is wrong in that regard and needs to be fixed.